### PR TITLE
T&D: Soft reset on PK non-null column change + More Snowflake Table info

### PR DESCRIPTION
**Pull request by evantahler**
This is a PR into https://github.com/airbytehq/airbyte/pull/30779 with some optional ideas.

https://github.com/airbytehq/airbyte/pull/30779 removes our explicit Type and Dedupe SQL to check that no records are missing primary keys and changes the format of our final tables to have a non-nullable columns.  This works great for tables created after this change.

We probably want to update the existing raw tables with a soft reset. The code in this PR works for BigQuery, as the BigQuery table definitions contain "mode" (required or not), but we don't have anything like this for Snowflake, and maybe other destinations as well.  So, we've got a few options:
1. Skip this work, as the blast radius is low.  We haven't seen many instances of records which lack a PK, and all new tables will be fine.  
2. Do this soft reset for only the destinations we can (e.g. Bigquery, not Snowflake)
3. **Update all destinations so we really do have the full table DDL so we can look at indexes and nullability.**. <-- we chose this

This PR will do a soft-reset of the existing raw tables, the first time, when merged

### CODE REVIEW

File: airbyte-integrations/bases/base-typing-deduping-test/src/main/java/io/airbyte/integrations/base/destination/typing_deduping/BaseSqlGeneratorIntegrationTest.java [hunk 1]
```diff
@@ -928,6 +929,41 @@ public void testReservedKeywords() throws Exception {
         dumpFinalTableRecords(streamId, ""));
   }
 
+  /**
+   * existingSchemaMatchesStreamConfig
+   */
+  @Test
+  public void ensurePKsAreIndexedUnique() throws Exception {
```
Comment (evantahler) [inline airbyte-integrations/bases/base-typing-deduping-test/src/main/java/io/airbyte/integrations/base/destination/typing_deduping/BaseSqlGeneratorIntegrationTest.java:936]: This new test succeeds for BigQuery and fails for Snowflake

File: airbyte-integrations/connectors/destination-snowflake/src/main/java/io/airbyte/integrations/destination/snowflake/typing_deduping/SnowflakeSqlGenerator.java [hunk 2]
```diff
@@ -146,12 +147,14 @@ public boolean existingSchemaMatchesStreamConfig(final StreamConfig stream, fina
         .collect(LinkedHashMap::new,
             (map, column) -> map.put(column.getKey(), column.getValue()),
             LinkedHashMap::putAll);
-    final boolean sameColumns = actualColumns.equals(intendedColumns)
+    final boolean countPksWithoutIndex = existingTable.columns().entrySet().stream()
+        .noneMatch(c -> pks.contains(c.getKey()) && !c.getValue().contains("NOT NULL"));
```
Comment (evantahler) [inline airbyte-integrations/connectors/destination-snowflake/src/main/java/io/airbyte/integrations/destination/snowflake/typing_deduping/SnowflakeSqlGenerator.java:151]: This doesn't work for snowflake - we don't currently have 'nullness' in `SnowflakeTableDefinition` ([link](https://github.com/airbytehq/airbyte/blob/master/airbyte-integrations/connectors/destination-snowflake/src/main/java/io/airbyte/integrations/destination/snowflake/typing_deduping/SnowflakeTableDefinition.java#L15))

**Comment by airbyte-oss-build-runner** [comment]
##  destination-bigquery test report (commit [`6f7bea9a7f`](https://github.com/airbytehq/airbyte/pull/30812/commits/6f7bea9a7f4462f259a558f7370bc73f55d044af)) - ✅

⏲️  Total pipeline duration: 07mn18s 

| Step                                                              | Result   |
|:------------------------------------------------------------------|:---------|
| Build connector tar                                               | ✅        |
| Build destination-bigquery docker image for platform linux/x86_64 | ✅        |
| Java Connector Unit Tests                                         | ✅        |
| Java Connector Integration Tests                                  | ✅        |
| Validate metadata for destination-bigquery                        | ✅        |
| Connector version semver check                                    | ✅        |
| Connector version increment check                                 | ✅        |
| QA checks                                                         | ✅        |

🔗 [View the logs here](https://storage.cloud.google.com/airbyte-ci-reports-multi/airbyte-ci/connectors/test/pull_request/evan_soft-rest-on-pl-index-change/1695859414/6f7bea9a7f4462f259a558f7370bc73f55d044af/destination-bigquery/2.0.20/output.html)

☁️ [View runs for commit in Dagger Cloud](https://alpha.dagger.cloud/changeByPipelines?filter=dagger.io/git.ref:6f7bea9a7f4462f259a558f7370bc73f55d044af)

*Please note that tests are only run on PR ready for review. Please set your PR to draft mode to not flood the CI engine and upstream service on following commits.*
**You can run the same pipeline locally on this branch with the [airbyte-ci](https://github.com/airbytehq/airbyte/blob/master/airbyte-ci/connectors/pipelines/README.md) tool with the following command**
```bash
airbyte-ci connectors --name=destination-bigquery test
```

**Comment by airbyte-oss-build-runner** [comment]
##  destination-snowflake test report (commit [`6f7bea9a7f`](https://github.com/airbytehq/airbyte/pull/30812/commits/6f7bea9a7f4462f259a558f7370bc73f55d044af)) - ❌

⏲️  Total pipeline duration: 01mn28s 

| Step                                        | Result   |
|:--------------------------------------------|:---------|
| Build connector tar                         | ❌        |
| Validate metadata for destination-snowflake | ✅        |
| Connector version semver check              | ✅        |
| Connector version increment check           | ✅        |
| QA checks                                   | ✅        |

🔗 [View the logs here](https://storage.cloud.google.com/airbyte-ci-reports-multi/airbyte-ci/connectors/test/pull_request/evan_soft-rest-on-pl-index-change/1695859414/6f7bea9a7f4462f259a558f7370bc73f55d044af/destination-snowflake/3.1.16/output.html)

☁️ [View runs for commit in Dagger Cloud](https://alpha.dagger.cloud/changeByPipelines?filter=dagger.io/git.ref:6f7bea9a7f4462f259a558f7370bc73f55d044af)

*Please note that tests are only run on PR ready for review. Please set your PR to draft mode to not flood the CI engine and upstream service on following commits.*
**You can run the same pipeline locally on this branch with the [airbyte-ci](https://github.com/airbytehq/airbyte/blob/master/airbyte-ci/connectors/pipelines/README.md) tool with the following command**
```bash
airbyte-ci connectors --name=destination-snowflake test
```

**Comment by airbyte-oss-build-runner** [comment]
##  destination-snowflake test report (commit [`c97241845d`](https://github.com/airbytehq/airbyte/pull/30812/commits/c97241845dfee610b3d02043c177ba960d4bba3f)) - ❌

⏲️  Total pipeline duration: 02mn09s 

| Step                                        | Result   |
|:--------------------------------------------|:---------|
| Build connector tar                         | ❌        |
| Validate metadata for destination-snowflake | ✅        |
| Connector version semver check              | ✅        |
| Connector version increment check           | ✅        |
| QA checks                                   | ✅        |

🔗 [View the logs here](https://storage.cloud.google.com/airbyte-ci-reports-multi/airbyte-ci/connectors/test/pull_request/evan_soft-rest-on-pl-index-change/1695861076/c97241845dfee610b3d02043c177ba960d4bba3f/destination-snowflake/3.1.16/output.html)

☁️ [View runs for commit in Dagger Cloud](https://alpha.dagger.cloud/changeByPipelines?filter=dagger.io/git.ref:c97241845dfee610b3d02043c177ba960d4bba3f)

*Please note that tests are only run on PR ready for review. Please set your PR to draft mode to not flood the CI engine and upstream service on following commits.*
**You can run the same pipeline locally on this branch with the [airbyte-ci](https://github.com/airbytehq/airbyte/blob/master/airbyte-ci/connectors/pipelines/README.md) tool with the following command**
```bash
airbyte-ci connectors --name=destination-snowflake test
```

**Comment by airbyte-oss-build-runner** [comment]
##  destination-bigquery test report (commit [`c97241845d`](https://github.com/airbytehq/airbyte/pull/30812/commits/c97241845dfee610b3d02043c177ba960d4bba3f)) - ✅

⏲️  Total pipeline duration: 03mn24s 

| Step                                                              | Result   |
|:------------------------------------------------------------------|:---------|
| Build connector tar                                               | ✅        |
| Build destination-bigquery docker image for platform linux/x86_64 | ✅        |
| Java Connector Unit Tests                                         | ✅        |
| Java Connector Integration Tests                                  | ✅        |
| Validate metadata for destination-bigquery                        | ✅        |
| Connector version semver check                                    | ✅        |
| Connector version increment check                                 | ✅        |
| QA checks                                                         | ✅        |

🔗 [View the logs here](https://storage.cloud.google.com/airbyte-ci-reports-multi/airbyte-ci/connectors/test/pull_request/evan_soft-rest-on-pl-index-change/1695861076/c97241845dfee610b3d02043c177ba960d4bba3f/destination-bigquery/2.0.20/output.html)

☁️ [View runs for commit in Dagger Cloud](https://alpha.dagger.cloud/changeByPipelines?filter=dagger.io/git.ref:c97241845dfee610b3d02043c177ba960d4bba3f)

*Please note that tests are only run on PR ready for review. Please set your PR to draft mode to not flood the CI engine and upstream service on following commits.*
**You can run the same pipeline locally on this branch with the [airbyte-ci](https://github.com/airbytehq/airbyte/blob/master/airbyte-ci/connectors/pipelines/README.md) tool with the following command**
```bash
airbyte-ci connectors --name=destination-bigquery test
```

**Comment by airbyte-oss-build-runner** [comment]
##  destination-snowflake test report (commit [`bc4149053a`](https://github.com/airbytehq/airbyte/pull/30812/commits/bc4149053a12a22dbfed9ae5908a5f204dadabba)) - ❌

⏲️  Total pipeline duration: 02mn00s 

| Step                                        | Result   |
|:--------------------------------------------|:---------|
| Build connector tar                         | ❌        |
| Validate metadata for destination-snowflake | ✅        |
| Connector version semver check              | ✅        |
| Connector version increment check           | ✅        |
| QA checks                                   | ✅        |

🔗 [View the logs here](https://storage.cloud.google.com/airbyte-ci-reports-multi/airbyte-ci/connectors/test/pull_request/evan_soft-rest-on-pl-index-change/1695913056/bc4149053a12a22dbfed9ae5908a5f204dadabba/destination-snowflake/3.1.16/output.html)

☁️ [View runs for commit in Dagger Cloud](https://alpha.dagger.cloud/changeByPipelines?filter=dagger.io/git.ref:bc4149053a12a22dbfed9ae5908a5f204dadabba)

*Please note that tests are only run on PR ready for review. Please set your PR to draft mode to not flood the CI engine and upstream service on following commits.*
**You can run the same pipeline locally on this branch with the [airbyte-ci](https://github.com/airbytehq/airbyte/blob/master/airbyte-ci/connectors/pipelines/README.md) tool with the following command**
```bash
airbyte-ci connectors --name=destination-snowflake test
```

**Comment by airbyte-oss-build-runner** [comment]
##  destination-bigquery test report (commit [`bc4149053a`](https://github.com/airbytehq/airbyte/pull/30812/commits/bc4149053a12a22dbfed9ae5908a5f204dadabba)) - ✅

⏲️  Total pipeline duration: 07mn52s 

| Step                                                              | Result   |
|:------------------------------------------------------------------|:---------|
| Build connector tar                                               | ✅        |
| Build destination-bigquery docker image for platform linux/x86_64 | ✅        |
| Java Connector Unit Tests                                         | ✅        |
| Java Connector Integration Tests                                  | ✅        |
| Validate metadata for destination-bigquery                        | ✅        |
| Connector version semver check                                    | ✅        |
| Connector version increment check                                 | ✅        |
| QA checks                                                         | ✅        |

🔗 [View the logs here](https://storage.cloud.google.com/airbyte-ci-reports-multi/airbyte-ci/connectors/test/pull_request/evan_soft-rest-on-pl-index-change/1695913056/bc4149053a12a22dbfed9ae5908a5f204dadabba/destination-bigquery/2.0.20/output.html)

☁️ [View runs for commit in Dagger Cloud](https://alpha.dagger.cloud/changeByPipelines?filter=dagger.io/git.ref:bc4149053a12a22dbfed9ae5908a5f204dadabba)

*Please note that tests are only run on PR ready for review. Please set your PR to draft mode to not flood the CI engine and upstream service on following commits.*
**You can run the same pipeline locally on this branch with the [airbyte-ci](https://github.com/airbytehq/airbyte/blob/master/airbyte-ci/connectors/pipelines/README.md) tool with the following command**
```bash
airbyte-ci connectors --name=destination-bigquery test
```

**Comment by airbyte-oss-build-runner** [comment]
##  destination-snowflake test report (commit [`4bc3d8adb0`](https://github.com/airbytehq/airbyte/pull/30812/commits/4bc3d8adb0957968d2261f5be852888a6f05d123)) - ❌

⏲️  Total pipeline duration: 04mn05s 

| Step                                        | Result   |
|:--------------------------------------------|:---------|
| Build connector tar                         | ❌        |
| Validate metadata for destination-snowflake | ✅        |
| Connector version semver check              | ✅        |
| Connector version increment check           | ✅        |
| QA checks                                   | ✅        |

🔗 [View the logs here](https://storage.cloud.google.com/airbyte-ci-reports-multi/airbyte-ci/connectors/test/pull_request/evan_soft-rest-on-pl-index-change/1695928570/4bc3d8adb0957968d2261f5be852888a6f05d123/destination-snowflake/3.1.17/output.html)

☁️ [View runs for commit in Dagger Cloud](https://alpha.dagger.cloud/changeByPipelines?filter=dagger.io/git.ref:4bc3d8adb0957968d2261f5be852888a6f05d123)

*Please note that tests are only run on PR ready for review. Please set your PR to draft mode to not flood the CI engine and upstream service on following commits.*
**You can run the same pipeline locally on this branch with the [airbyte-ci](https://github.com/airbytehq/airbyte/blob/master/airbyte-ci/connectors/pipelines/README.md) tool with the following command**
```bash
airbyte-ci connectors --name=destination-snowflake test
```

**Comment by airbyte-oss-build-runner** [comment]
##  destination-bigquery test report (commit [`4bc3d8adb0`](https://github.com/airbytehq/airbyte/pull/30812/commits/4bc3d8adb0957968d2261f5be852888a6f05d123)) - ✅

⏲️  Total pipeline duration: 09mn40s 

| Step                                                              | Result   |
|:------------------------------------------------------------------|:---------|
| Build connector tar                                               | ✅        |
| Build destination-bigquery docker image for platform linux/x86_64 | ✅        |
| Java Connector Unit Tests                                         | ✅        |
| Java Connector Integration Tests                                  | ✅        |
| Validate metadata for destination-bigquery                        | ✅        |
| Connector version semver check                                    | ✅        |
| Connector version increment check                                 | ✅        |
| QA checks                                                         | ✅        |

🔗 [View the logs here](https://storage.cloud.google.com/airbyte-ci-reports-multi/airbyte-ci/connectors/test/pull_request/evan_soft-rest-on-pl-index-change/1695928570/4bc3d8adb0957968d2261f5be852888a6f05d123/destination-bigquery/2.0.20/output.html)

☁️ [View runs for commit in Dagger Cloud](https://alpha.dagger.cloud/changeByPipelines?filter=dagger.io/git.ref:4bc3d8adb0957968d2261f5be852888a6f05d123)

*Please note that tests are only run on PR ready for review. Please set your PR to draft mode to not flood the CI engine and upstream service on following commits.*
**You can run the same pipeline locally on this branch with the [airbyte-ci](https://github.com/airbytehq/airbyte/blob/master/airbyte-ci/connectors/pipelines/README.md) tool with the following command**
```bash
airbyte-ci connectors --name=destination-bigquery test
```

**Comment by airbyte-oss-build-runner** [comment]
##  destination-bigquery test report (commit [`896cbdab99`](https://github.com/airbytehq/airbyte/pull/30812/commits/896cbdab994f21aa0bf35e8b686e4ecb91e2c01d)) - ✅

⏲️  Total pipeline duration: 08mn41s 

| Step                                                              | Result   |
|:------------------------------------------------------------------|:---------|
| Build connector tar                                               | ✅        |
| Build destination-bigquery docker image for platform linux/x86_64 | ✅        |
| Java Connector Unit Tests                                         | ✅        |
| Java Connector Integration Tests                                  | ✅        |
| Validate metadata for destination-bigquery                        | ✅        |
| Connector version semver check                                    | ✅        |
| Connector version increment check                                 | ✅        |
| QA checks                                                         | ✅        |

🔗 [View the logs here](https://storage.cloud.google.com/airbyte-ci-reports-multi/airbyte-ci/connectors/test/pull_request/evan_soft-rest-on-pl-index-change/1695943536/896cbdab994f21aa0bf35e8b686e4ecb91e2c01d/destination-bigquery/2.0.20/output.html)

☁️ [View runs for commit in Dagger Cloud](https://alpha.dagger.cloud/changeByPipelines?filter=dagger.io/git.ref:896cbdab994f21aa0bf35e8b686e4ecb91e2c01d)

*Please note that tests are only run on PR ready for review. Please set your PR to draft mode to not flood the CI engine and upstream service on following commits.*
**You can run the same pipeline locally on this branch with the [airbyte-ci](https://github.com/airbytehq/airbyte/blob/master/airbyte-ci/connectors/pipelines/README.md) tool with the following command**
```bash
airbyte-ci connectors --name=destination-bigquery test
```

**Comment by airbyte-oss-build-runner** [comment]
##  destination-snowflake test report (commit [`b4ac6af945`](https://github.com/airbytehq/airbyte/pull/30812/commits/b4ac6af945763f9d61e09abeef0a1604ebfc4138)) - ✅

⏲️  Total pipeline duration: 16mn52s 

| Step                                                               | Result   |
|:-------------------------------------------------------------------|:---------|
| Build connector tar                                                | ✅        |
| Build destination-snowflake docker image for platform linux/x86_64 | ✅        |
| Java Connector Unit Tests                                          | ✅        |
| Java Connector Integration Tests                                   | ✅        |
| Validate metadata for destination-snowflake                        | ✅        |
| Connector version semver check                                     | ✅        |
| Connector version increment check                                  | ✅        |
| QA checks                                                          | ✅        |

🔗 [View the logs here](https://storage.cloud.google.com/airbyte-ci-reports-multi/airbyte-ci/connectors/test/pull_request/evan_soft-rest-on-pl-index-change/1696004325/b4ac6af945763f9d61e09abeef0a1604ebfc4138/destination-snowflake/3.1.17/output.html)

☁️ [View runs for commit in Dagger Cloud](https://alpha.dagger.cloud/changeByPipelines?filter=dagger.io/git.ref:b4ac6af945763f9d61e09abeef0a1604ebfc4138)

*Please note that tests are only run on PR ready for review. Please set your PR to draft mode to not flood the CI engine and upstream service on following commits.*
**You can run the same pipeline locally on this branch with the [airbyte-ci](https://github.com/airbytehq/airbyte/blob/master/airbyte-ci/connectors/pipelines/README.md) tool with the following command**
```bash
airbyte-ci connectors --name=destination-snowflake test
```

**Comment by airbyte-oss-build-runner** [comment]
##  destination-snowflake test report (commit [`2d50820699`](https://github.com/airbytehq/airbyte/pull/30812/commits/2d50820699a7c397521f8991985b986a70b0a011)) - ✅

⏲️  Total pipeline duration: 09mn22s 

| Step                                                               | Result   |
|:-------------------------------------------------------------------|:---------|
| Build connector tar                                                | ✅        |
| Build destination-snowflake docker image for platform linux/x86_64 | ✅        |
| Java Connector Unit Tests                                          | ✅        |
| Java Connector Integration Tests                                   | ✅        |
| Validate metadata for destination-snowflake                        | ✅        |
| Connector version semver check                                     | ✅        |
| Connector version increment check                                  | ✅        |
| QA checks                                                          | ✅        |

🔗 [View the logs here](https://storage.cloud.google.com/airbyte-ci-reports-multi/airbyte-ci/connectors/test/pull_request/evan_soft-rest-on-pl-index-change/1696005764/2d50820699a7c397521f8991985b986a70b0a011/destination-snowflake/3.1.17/output.html)

☁️ [View runs for commit in Dagger Cloud](https://alpha.dagger.cloud/changeByPipelines?filter=dagger.io/git.ref:2d50820699a7c397521f8991985b986a70b0a011)

*Please note that tests are only run on PR ready for review. Please set your PR to draft mode to not flood the CI engine and upstream service on following commits.*
**You can run the same pipeline locally on this branch with the [airbyte-ci](https://github.com/airbytehq/airbyte/blob/master/airbyte-ci/connectors/pipelines/README.md) tool with the following command**
```bash
airbyte-ci connectors --name=destination-snowflake test
```

**Comment by airbyte-oss-build-runner** [comment]
##  destination-bigquery test report (commit [`2d50820699`](https://github.com/airbytehq/airbyte/pull/30812/commits/2d50820699a7c397521f8991985b986a70b0a011)) - ❌

⏲️  Total pipeline duration: 01mn56s 

| Step                                       | Result   |
|:-------------------------------------------|:---------|
| Build connector tar                        | ❌        |
| Validate metadata for destination-bigquery | ✅        |
| Connector version semver check             | ✅        |
| Connector version increment check          | ✅        |
| QA checks                                  | ✅        |

🔗 [View the logs here](https://storage.cloud.google.com/airbyte-ci-reports-multi/airbyte-ci/connectors/test/pull_request/evan_soft-rest-on-pl-index-change/1696005764/2d50820699a7c397521f8991985b986a70b0a011/destination-bigquery/2.0.20/output.html)

☁️ [View runs for commit in Dagger Cloud](https://alpha.dagger.cloud/changeByPipelines?filter=dagger.io/git.ref:2d50820699a7c397521f8991985b986a70b0a011)

*Please note that tests are only run on PR ready for review. Please set your PR to draft mode to not flood the CI engine and upstream service on following commits.*
**You can run the same pipeline locally on this branch with the [airbyte-ci](https://github.com/airbytehq/airbyte/blob/master/airbyte-ci/connectors/pipelines/README.md) tool with the following command**
```bash
airbyte-ci connectors --name=destination-bigquery test
```

**Comment by airbyte-oss-build-runner** [comment]
##  destination-bigquery test report (commit [`24dc8f1fd7`](https://github.com/airbytehq/airbyte/pull/30812/commits/24dc8f1fd7ce62f3ef83d315f6b9ad9c48098284)) - ❌

⏲️  Total pipeline duration: 03mn14s 

| Step                                       | Result   |
|:-------------------------------------------|:---------|
| Build connector tar                        | ❌        |
| Validate metadata for destination-bigquery | ✅        |
| Connector version semver check             | ✅        |
| Connector version increment check          | ✅        |
| QA checks                                  | ✅        |

🔗 [View the logs here](https://storage.cloud.google.com/airbyte-ci-reports-multi/airbyte-ci/connectors/test/pull_request/evan_soft-rest-on-pl-index-change/1696006982/24dc8f1fd7ce62f3ef83d315f6b9ad9c48098284/destination-bigquery/2.0.20/output.html)

☁️ [View runs for commit in Dagger Cloud](https://alpha.dagger.cloud/changeByPipelines?filter=dagger.io/git.ref:24dc8f1fd7ce62f3ef83d315f6b9ad9c48098284)

*Please note that tests are only run on PR ready for review. Please set your PR to draft mode to not flood the CI engine and upstream service on following commits.*
**You can run the same pipeline locally on this branch with the [airbyte-ci](https://github.com/airbytehq/airbyte/blob/master/airbyte-ci/connectors/pipelines/README.md) tool with the following command**
```bash
airbyte-ci connectors --name=destination-bigquery test
```

**Comment by airbyte-oss-build-runner** [comment]
##  destination-snowflake test report (commit [`24dc8f1fd7`](https://github.com/airbytehq/airbyte/pull/30812/commits/24dc8f1fd7ce62f3ef83d315f6b9ad9c48098284)) - ✅

⏲️  Total pipeline duration: 03mn10s 

| Step                                                               | Result   |
|:-------------------------------------------------------------------|:---------|
| Build connector tar                                                | ✅        |
| Build destination-snowflake docker image for platform linux/x86_64 | ✅        |
| Java Connector Unit Tests                                          | ✅        |
| Java Connector Integration Tests                                   | ✅        |
| Validate metadata for destination-snowflake                        | ✅        |
| Connector version semver check                                     | ✅        |
| Connector version increment check                                  | ✅        |
| QA checks                                                          | ✅        |

🔗 [View the logs here](https://storage.cloud.google.com/airbyte-ci-reports-multi/airbyte-ci/connectors/test/pull_request/evan_soft-rest-on-pl-index-change/1696006982/24dc8f1fd7ce62f3ef83d315f6b9ad9c48098284/destination-snowflake/3.1.17/output.html)

☁️ [View runs for commit in Dagger Cloud](https://alpha.dagger.cloud/changeByPipelines?filter=dagger.io/git.ref:24dc8f1fd7ce62f3ef83d315f6b9ad9c48098284)

*Please note that tests are only run on PR ready for review. Please set your PR to draft mode to not flood the CI engine and upstream service on following commits.*
**You can run the same pipeline locally on this branch with the [airbyte-ci](https://github.com/airbytehq/airbyte/blob/master/airbyte-ci/connectors/pipelines/README.md) tool with the following command**
```bash
airbyte-ci connectors --name=destination-snowflake test
```

**Review by jbfbell — APPROVED** [review]

**Comment by airbyte-oss-build-runner** [comment]
##  destination-snowflake test report (commit [`0609493302`](https://github.com/airbytehq/airbyte/pull/30812/commits/060949330216d0a521f57d7388883860523472ba)) - ✅

⏲️  Total pipeline duration: 08mn59s 

| Step                                                               | Result   |
|:-------------------------------------------------------------------|:---------|
| Build connector tar                                                | ✅        |
| Build destination-snowflake docker image for platform linux/x86_64 | ✅        |
| Java Connector Unit Tests                                          | ✅        |
| Java Connector Integration Tests                                   | ✅        |
| Validate metadata for destination-snowflake                        | ✅        |
| Connector version semver check                                     | ✅        |
| Connector version increment check                                  | ✅        |
| QA checks                                                          | ✅        |

🔗 [View the logs here](https://storage.cloud.google.com/airbyte-ci-reports-multi/airbyte-ci/connectors/test/pull_request/evan_soft-rest-on-pl-index-change/1696008124/060949330216d0a521f57d7388883860523472ba/destination-snowflake/3.1.17/output.html)

☁️ [View runs for commit in Dagger Cloud](https://alpha.dagger.cloud/changeByPipelines?filter=dagger.io/git.ref:060949330216d0a521f57d7388883860523472ba)

*Please note that tests are only run on PR ready for review. Please set your PR to draft mode to not flood the CI engine and upstream service on following commits.*
**You can run the same pipeline locally on this branch with the [airbyte-ci](https://github.com/airbytehq/airbyte/blob/master/airbyte-ci/connectors/pipelines/README.md) tool with the following command**
```bash
airbyte-ci connectors --name=destination-snowflake test
```

**Comment by airbyte-oss-build-runner** [comment]
##  destination-bigquery test report (commit [`0609493302`](https://github.com/airbytehq/airbyte/pull/30812/commits/060949330216d0a521f57d7388883860523472ba)) - ✅

⏲️  Total pipeline duration: 08mn12s 

| Step                                                              | Result   |
|:------------------------------------------------------------------|:---------|
| Build connector tar                                               | ✅        |
| Build destination-bigquery docker image for platform linux/x86_64 | ✅        |
| Java Connector Unit Tests                                         | ✅        |
| Java Connector Integration Tests                                  | ✅        |
| Validate metadata for destination-bigquery                        | ✅        |
| Connector version semver check                                    | ✅        |
| Connector version increment check                                 | ✅        |
| QA checks                                                         | ✅        |

🔗 [View the logs here](https://storage.cloud.google.com/airbyte-ci-reports-multi/airbyte-ci/connectors/test/pull_request/evan_soft-rest-on-pl-index-change/1696008124/060949330216d0a521f57d7388883860523472ba/destination-bigquery/2.0.20/output.html)

☁️ [View runs for commit in Dagger Cloud](https://alpha.dagger.cloud/changeByPipelines?filter=dagger.io/git.ref:060949330216d0a521f57d7388883860523472ba)

*Please note that tests are only run on PR ready for review. Please set your PR to draft mode to not flood the CI engine and upstream service on following commits.*
**You can run the same pipeline locally on this branch with the [airbyte-ci](https://github.com/airbytehq/airbyte/blob/master/airbyte-ci/connectors/pipelines/README.md) tool with the following command**
```bash
airbyte-ci connectors --name=destination-bigquery test
```

**Comment by evantahler** [comment]
Thank you @jbfbell!

**Pull request merged**